.kokoro/docs.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -xeo pipefail
```

Comment:
this looks like a slightly older version of config then is currently live in `synthtool`. We dropped `-xeo` and went with `-eo`, because `-x` leaks a lot of environment information.

Reply:
I see. I used the stable build of synthtool. Sure enough with latest the contents have now changed.

Reply:
I'll see about getting a new version of synthtool out the door.